core/rpc_invoker.go

```diff
@@ -77,6 +79,33 @@ func (ri *RPCInvoker) Invoke(ctx context.Context, microServiceName, schemaID, op
 	i.Args = arg
 	i.Reply = reply
 	i.Ctx = ctx
+	return ri.invoke(i, reply)
+}
+
+// abstract invoker is a common invoker for RPC
```

Comment:
if it is invoker for both rest and rpc then put is in isolated go file and change comment

Reply:
Done.

Reply:
Done.